ow_power_system/config/system.cfg

```diff
@@ -28,16 +28,21 @@ voltage_range: 0.1			        # volts
 # values from other planetary rovers, and some are just guesses.
 
 # Baseline values (watts) for constant power draws.
-baseline_power_camera: 0.1
-baseline_power_communications: 0.6
 baseline_power_computing: 0.1
 baseline_power_heating: 1.2
 baseline_power_science_instr: 0.6
 baseline_power_sample_handling: 0.6
 baseline_power_other: 2.2
 
+
 # Power coefficients for dynamically computed power draw.
-wattage_coefficient_lights: 0.3    # power draw (W) per light at max intensity
+power_lights_coefficient: 0.3       # power draw (W) per light at max intensity
```

Comment:
Although I introduced the `coefficient` suffix, I'm not sure I like it any more.  Would `power_lights` be better?

Reply:
Now changed to `power_active_lights`